sample_apps/simple_app_python/example_app.py

```diff
@@ -277,28 +270,71 @@ def server_dispatch(data_dir, server_app_host, server_app_port):
     return True
 
 ###############################################################################
-def client_dispatch(data_dir, server_app_host, server_app_port):
-    """
-    Start a client app that sends a message via secure SSL connection.
+def setup_server_ssl_context(cctm, data_dir, print_all):
     """
-    print(fnl(), 'Start client application process ...\n')
+    Setup a SSL context for server-process, verifying certificates v/s
+    root certificate issued by CA.
 
-    # Client needs to authenticate the server; hence 'Purpose.SERVER_AUTH'
-    context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
+    **** NOTE: **** SSL interfaces need to read the private key from a file.
+
+    So, this method invokes a Certifier trust manager's method to persist
```

Comment:
This setup method is where we persist private-key data from trust_manager to [a] named temporary file, in an anonymous-but-named temp-dir.